Formula/openssl@3.rb

```diff
@@ -1,16 +1,16 @@
-class OpensslAT11 < Formula
+class OpensslAT3 < Formula
   desc "Cryptography and SSL/TLS Toolkit"
   homepage "https://openssl.org/"
-  url "https://www.openssl.org/source/openssl-1.1.1l.tar.gz"
-  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.1.1l.tar.gz"
-  mirror "https://www.openssl.org/source/old/1.1.1/openssl-1.1.1l.tar.gz"
-  sha256 "0b7a3e5e59c34827fe0c3a74b7ec8baef302b98fa80088d7f9153aa16fa76bd1"
-  license "OpenSSL"
+  url "https://www.openssl.org/source/openssl-3.0.0.tar.gz"
+  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-3.0.0.tar.gz"
+  mirror "https://www.openssl.org/source/old/3.0/openssl-3.0.0.tar.gz"
+  sha256 "59eedfcb46c25214c9bd37ed6078297b4df01d012267fe9e9eee31f61bc70536"
+  license "Apache-2.0"
   version_scheme 1
 
   livecheck do
     url "https://www.openssl.org/source/"
-    regex(/href=.*?openssl[._-]v?(1\.1(?:\.\d+)+[a-z]?)\.t/i)
+    regex(/href=.*?openssl-(\d+(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?openssl[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

This regex works but it could use a couple tweaks to bring it back in line with livecheck regex standards. Don't feel the need to update this PR for such a minor change but I figured I would comment while I was passing through, in case this will be updated again anyway. Otherwise, I'll take care of it in a follow-up PR.